docs/ethical-advertising.rst

```diff
@@ -0,0 +1,123 @@
+Ethical Advertising
+===================
+
+Read the Docs is a large free website on the internet.
```

Comment:
`a large free website on the internet` is nondescript, explain in more detail or omit this.